Rakefile

```diff
@@ -48,8 +167,69 @@ namespace :test do
     end
     Dir.chdir( original_dir )
   end
+
+  desc <<-EOS
+Run the base beaker acceptance tests
+#{USAGE}
+  EOS
+  task :base  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/base'})
+  end
+
+  desc <<-EOS
+Run the hypervisor beaker acceptance tests
+#{USAGE}
+  EOS
+  task :hypervisor  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/hypervisor'})
+  end
+
+  desc <<-EOS
+Run the puppet beaker acceptance tests on a pe install.
+#{USAGE}
+  EOS
+  task :puppetpe  => 'gen_hosts' do
+    beaker_test({:pre_suite => 'acceptance/pre_suite/pe/install.rb', :tests => 'acceptance/tests/puppet'})
+  end
+
+  desc <<-EOS
+Run the puppet beaker acceptance tests on a puppet gem install.
+#{USAGE}
+  EOS
+  task :puppetgem  => 'gen_hosts' do
+    beaker_test({:pre_suite => 'acceptance/pre_suite/puppet_gem/install.rb', :tests => 'acceptance/tests/puppet'})
+  end
+
+  desc <<-EOS
+Run the puppet beaker acceptance tests on a puppet git install.
+#{USAGE}
+  EOS
+  task :puppetgit  => 'gen_hosts' do
+    beaker_test({:pre_suite => 'acceptance/pre_suite/puppet_git/install.rb', :tests => 'acceptance/tests/puppet'})
+  end
+
+  desc <<-EOS
+Run the puppet beaker acceptance tests on a puppet package install.
+#{USAGE}
+  EOS
+  task :puppetpkg => 'gen_hosts' do
+    beaker_test({:pre_suite => 'acceptance/pre_suite/puppet_pkg/install.rb', :tests => 'acceptance/tests/puppet'})
+  end
+
+  desc 'Generate Beaker Host Config File'
+  task :gen_hosts do
+    if hosts_file_env
+      next
+    end
+    cli = BeakerHostGenerator::CLI.new([test_targets, "--pe_dir=#{pe_dir}"])
```

Comment:
should this --pe-dir flag be conditional?  it's being specified for all test types, it appears.